tests/AUTOMATE_TEST_GUIDE.md

```diff
@@ -0,0 +1,17 @@
+# Create new autoamted test case - Guide
+
+A structure that can be used to have an idea about the creation of a new automated test case.
```

Comment:
Although I understand what you are saying here, I think it could help if this introduction changes a bit. Perhaps, similar to what we have for the [documentation style guide](https://docs.communityhealthtoolkit.org/contribute/docs/style-guide/). So, I suggest something like:

"This style guide provides editorial guidelines for anyone creating new automated test cases for the CHT-Core."

That is just an example; I'm not suggesting that's what it should say exactly. But, the point is, when someone reads that, they can see it as "oh, this is what I need to do, this is my reference from now on".